reinforcement_learning/include/trace_logger.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <string>
```

Comment:
string could be forward declared here if we'd prefer

Reply:
string is a template class and forward declaration has issues.